packages/atproto_crypto/algs/base_alg.py

```diff
@@ -34,11 +35,23 @@ def decompress_pubkey(self, pubkey: bytes) -> bytes:
             encoding=serialization.Encoding.X962, format=serialization.PublicFormat.UncompressedPoint
         )
 
-    @staticmethod
-    def _encode_signature(signature: bytes) -> bytes:
+    def _validate_dss_signature_s(self, s: int) -> None:
```

Comment:
Just as a code style thing, I'd recommend giving this function a more explicit name, e.g. `_ensure_dss_signature_low_s`. The word "validate" could mislead someone into thinking it does full signature verification, as opposed to just the low-s check.

Reply:
renamed!